tests/build-delete.sh

```diff
@@ -0,0 +1,56 @@
+source common.sh
+
+clearStore
+
+set -o pipefail
+
+# https://github.com/NixOS/nix/issues/6572
```

Comment:
These tests do not yet work with CA derivations, so I pulled them out into their own file.